plugins/plugin-java-debugger/che-plugin-java-debugger-server/src/test/resources/testclasses/ThreadDumpTest.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
This looks more like an experiment someone did than a test. Do we need it?

Reply:
Test classes for java debugger server side tests.
We do need them.